app.js

```diff
@@ -1,2 +1,31 @@
-const BOARD = document.getElementById('letters');
+const gameWrapper = document.querySelector('.game-wrapper');
+gameSetup();
 
+function gameSetup() {
+  for (i = 0; i < 26; i++) {
+    let insertedDiv = document.createElement('div');
+    let paragraph = document.createElement('p');
+    let letter = String.fromCharCode(i + 97);
+    paragraph.innerHTML = letter;
+    insertedDiv.append(paragraph);
+    insertedDiv.classList.add('letter-wrapper', `${letter}`, 'border-all');
```

Comment:
to have a prettier code, you could extract all of the class work to separate function which would be invoked here.